CHANGELOG

```diff
@@ -1,6 +1,8 @@
 devel
 -----
 
+* issue #1190: added option `--create-database` for arangoimp
```

Comment:
note: in devel we have renamed arangoimp to arangoimport.
arangoimp should just be a symlink to arangoimport in devel.